doc/Changelog.md

```diff
@@ -13,6 +15,14 @@ Changelog {#Changelog}
   - brain::Circuit::getGIDs( target ) returns GIDS for a specified target.
   - brain::Circuit::getNumNeurons() returns the total number of neurons in the
     circuit.
+* [43](https://github.com/BlueBrain/Brion/pull/43):
+  Add MVD3 support to Brain::circuit 
+  - Add mvd-tool as a subroject for MVD3 parsing
+  - Change brain::getTransforms logic from euler angles to rotation with quaternions
+  - Add switch in BlueConfig to load or not MVD3
+  - Add tests for MVD3 parsing
+  - Add Nix support
+  - Switch MVDTool dependency to github repository
```

Comment:
Too long. This is a Changelog, if I want details on a change I go to the PR. Just remove the bullet points.

brain is lowercase (namespace)